libs/core/src/lib/nested-list/nested-list-directives.ts

```diff
@@ -36,21 +40,63 @@ export class NestedListIconDirective extends AbstractFdNgxClass {
     constructor(private elementRef: ElementRef) {
         super(elementRef);
     }
+
 }
 
 @Directive({
     selector: '[fdNestedDirectivesTitle], [fd-nested-list-title]'
 })
 export class NestedListTitleDirective {
+
     /** @hidden */
     @HostBinding('class.fd-nested-list__title')
     fdNestedListTitleClass: boolean = true;
 
     /** @hidden */
-    constructor(private elementRef: ElementRef) {}
+    constructor(
+        private elementRef: ElementRef
+    ) {
+    }
 
     /** Returns element's InnerText */
     public getInnerText(): string {
         return this.elementRef && this.elementRef.nativeElement.innerText;
     }
 }
+
+@Directive({
+    selector: '[fdNestedListExpandIcon], [fd-nested-list-expand-icon]',
+    host: {
+        'aria-haspopup': 'true'
+    }
+})
+export class NestedListExpandIconDirective {
+
+    /** @hidden */
+    @HostBinding('class.fd-nested-list__expand-icon')
+    fdNestedListTitleClass: boolean = true;
+
+    /**
+     * @hidden
+     * Attribute controlled by the parent `NestedItemDirective`
+     */
+    @HostBinding('class.is-expanded')
+    @HostBinding('attr.aria-expanded')
+    expanded: boolean = false;
+
+    /** Event thrown, when icon is clicked */
+    @Output()
+    clicked: EventEmitter<MouseEvent> = new EventEmitter<MouseEvent>();
```

Comment:
What is the value of this `@Output()`? User can set `(click)` by himself.

Reply:
It's quite useless